definition.sql

```diff
@@ -0,0 +1,8 @@
+CREATE DATABASE memoapp ENCODING = 'UTF8' TEMPLATE template0;
+
+CREATE TABLE memo
+(id serial not null,
```

Comment:
```suggestion
(id SERIAL NOT NULL,
```

DBの予約語を全部大文字にすると識別子と区別しやすくなります。